redis/config.go

```diff
@@ -55,5 +56,9 @@ func (r *Config) Validate() error {
 		return errors.New("Redis host missing")
 	}
 
+	if r.Username != "" && r.Password == "" {
+		return errors.New("A Redis username requires a password as well")
```

Comment:
GO errors always begin in lower case, so I would rephrase this to `'password' must be set, if username is provided` and get rid of the `A ...` part.

Reply:
Good catch, thanks.

Reply:
I think we should always put important information at the beginning of the sentence. The user has to perform an action, so I opt for Yonas' suggestion.